application.cpp

```diff
@@ -0,0 +1,23 @@
+//
+// Created by kirill on 14.04.2020.
+//
+
+#include "application.h"
+
+Application::Application(const std::string& file_in, const std::string& file_out, int unit_size)
```

Comment:
the alignment of the whole structure is wrong, but it can be easily fixed by the file watcher in the clion